### PR TITLE
Adding Python Code

### PR DESCRIPTION
**Pull request by DHCJS**

### CODE REVIEW

**Review by DHCJS — COMMENTED** [review]
I've changed into Python, before was HTML

File: contribution/mycode.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+temp = "6 degrees"
```
Comment (DHCJS) [inline contribution/mycode.py:1]: Is this the correct way to write Python? I just learn it from Freecodecamp

Comment (noahcoder11) [reply contribution/mycode.py:1]: For this purpose, I would say to not make it a string. If you want it to be a float, then you can say `temp = 6.0`, then maybe add a comment that lets the person reading the code know that it is in degrees Fahrenheit

Comment (DHCJS) [reply contribution/mycode.py]: Thank you for the information!

**Comment by Faiznurullah** [comment]
please make in your folder projects

**Comment by Faiznurullah** [comment]
Read rule repo

**Comment by DHCJS** [comment]
> please make in your folder projects

Done, please kindly check it.

**Comment by Faiznurullah** [comment]
greet

**Pull request merged**